<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleReportWidget</class>
 <widget class="QWidget" name="SimpleReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formSimpleReport">
         <item row="0" column="0">
          <widget class="QLabel" name="lblTitle">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editTitle"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblHeader">
           <property name="text">
            <string>Header</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="XTextEdit" name="textHeader"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblFooter">
           <property name="text">
            <string>Footer</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="XTextEdit" name="textFooter"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="toolsLayout">
         <item>
          <widget class="QToolButton" name="setTblHeadersButton">
           <property name="toolTip">
            <string>Set Table Headers</string>
           </property>
           <property name="text">
            <string>Rename Columns</string>
           </property>
           <property name="icon">
            <iconset theme="insert-text">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="printButton">
           <property name="toolTip">
            <string>Print Report</string>
           </property>
           <property name="text">
            <string>Print Report</string>
           </property>
           <property name="icon">
            <iconset theme="printer">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="previewButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="exportButton">
           <property name="toolTip">
            <string>Export Report to PDF</string>
           </property>
           <property name="text">
            <string>pdf</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="clearSrPropertiesButton">
           <property name="toolTip">
            <string>Clear Report Fields</string>
           </property>
           <property name="text">
            <string>Clear Fields</string>
           </property>
           <property name="icon">
            <iconset theme="edit-clear">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="querySrTable">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XTextEdit</class>
   <extends>QWidget</extends>
   <header>xtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
